astroplan/tests/test_core.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
```

Comment:
As far as I know it's common practice to use `from astropy.tests.helper import remote_data` in affiliated packages.
E.g. [gammapy](https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=remote_data) and [astroquery](https://github.com/astropy/astroquery/search?utf8=%E2%9C%93&q=remote_data) do this.

But `remote_data` does not show up in the API reference docs for Astropy:
http://astropy.readthedocs.org/en/latest/search.html?q=remote_data&check_keywords=yes&area=default

I think `remote_data` should be part of Astropy's public API and show up in the docs.

@eteq – What do you think?